### PR TITLE
Expeditionary Forces Arms Dealer

### PR DESCRIPTION
**Pull request by Silence-112**
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement
4. [ ] Miscellaneous

### What have you changed and why?
Information:

Added arms dealer vehicles and weapons/equipment from [Expeditionary Forces](https://store.steampowered.com/app/2647830/Arma_3_Creator_DLC_Expeditionary_Forces/)

### Please specify which Issue this PR Resolves (If Applicable).
"This PR closes #XXXX!"

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

### CODE REVIEW

**Comment by wersal454** [comment]
everything is fine, missed the heli

Also I would suggest swapping combat boats for their AAF(independent) analogs

File: A3A/addons/ultimate/config/trader/vehicles/vehicles_ef.hpp
Comment (wersal454) [inline A3A/addons/ultimate/config/trader/vehicles/vehicles_ef.hpp:1]: new heli pls

Comment (Silence-112) [reply A3A/addons/ultimate/config/trader/vehicles/vehicles_ef.hpp:1]: Add it to the sheet

Comment (wersal454) [reply A3A/addons/ultimate/config/trader/vehicles/vehicles_ef.hpp:1]: you've added it already
image
but you didn't make the calculations

**Pull request merged**